src/draw.c

```diff
@@ -120,6 +125,40 @@ static struct color string_to_color(const char *str)
         return hex_to_color(0xF, 1);
 }
 
+static struct gradient string_to_gradient(char **strv)
+{
+        struct gradient g;
+        g.length = 0;
+
+        if (strv == NULL) {
+                LOG_W("Invalid empty color gradient string");
+                g.cs = g_malloc(sizeof(struct color));
+                g.cs[g.length++] = hex_to_color(0xF, 1);
+                /* return black (only one color) on error */
+        } else {
+                g.cs = g_malloc0(g_strv_length(strv) * sizeof(struct color));
+                for (int i = 0; strv[i] != NULL; i++) {
+                        char *end, *str = strv[i];
+                        if (STR_EMPTY(str)) continue;
```

Comment:
Just for understanding: this would mean an empty element would be allowed and yield a opaque black?

Reply:
I am following the previous behavior